readme.md

```diff
@@ -0,0 +1,165 @@
+# WP Endpoint Collection
+
+> Query against a WordPress site and get a JSON response with a
+> collection of data associated with your request.
+
+## Getting Started
+
+The easiest way to install this package is by using composer from your terminal:
+
+```bash
+composer require moxie-lean/wp-endpoints-archive
```

Comment:
Same as before.

> On Mar 31, 2016, at 6:59 AM, Adam Fenton <notifications@github.com> wrote:
> 
> In readme.md <https://github.com/moxie-leean/wp-endpoints-collection/pull/2#discussion_r58048633>:
> 
> > @@ -0,0 +1,165 @@
> > +# WP Endpoint Collection
> > +
> > +> Query against a WordPress site and get a JSON response with a
> > +> collection of data associated with your request.
> > +
> > +## Getting Started
> > +
> > +The easiest way to install this package is by using composer from your terminal:
> > +
> > +```bash
> > +composer require moxie-lean/wp-endpoints-archive
> Why are you using the name archive still?
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub <https://github.com/moxie-leean/wp-endpoints-collection/pull/2/files/79212b057c9e0b6231ca448bc9f4e34930f2b65e#r58048633>